src/main.rs

```diff
@@ -42,9 +53,12 @@ fn main() {
             TransferCharacteristic::SRGB,
             ColorPrimaries::BT709,
         )
-        .unwrap(),
+        .expect("Failed to process distorted_data into RGB"),
     )
-    .unwrap();
-    let result = compute_frame_ssimulacra2(source_data, distorted_data).unwrap();
-    eprintln!("{:.8}", result);
+    .expect("Failed to process distorted_data into XYB");
+
+    let result = compute_frame_ssimulacra2(source_data, distorted_data)
+        .expect("Failed to calculate ssimulacra2");
+
+    println!("{result:.8}");
```

Comment:
```suggestion
    eprintln!("{result:.8}");
```

Reply:
Actually this probably should have been `println`. My original code of `eprintln` doesn't follow standard convention, the result should go to stdout.